python/dalex/dalex/_arena/_plot_container.py

```diff
@@ -0,0 +1,61 @@
+from .params import ModelParam, DatasetParam, VariableParam, ObservationParam
+
+class PlotContainer:
+    def __init__(self, arena):
+        if type(arena).__name__ != 'Arena' or type(arena).__module__ != 'dalex._arena.object':
+            raise Exception('Invalid Arena argument')
+        info = self.__class__.info
```

Comment:
curious: Is there any point in using `__class__` ? e.g.

![image](https://user-images.githubusercontent.com/32574004/98451586-6cff5700-2147-11eb-8b38-5e93ea2b04be.png)

Reply:
Thanks for that. I never thought these variables can be accessed without __class__.